### PR TITLE
[netatmo] Additional examples for home and camera

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@jlaur @kaikreuzer : this is only documentation with a new example covering home and camera that could be helpful for users using config files. I would suggest to merge this change before releasing 3.3.

File: bundles/org.openhab.binding.netatmo/README.md [hunk 1]
```diff
@@ -652,72 +652,83 @@ All these channels are read only.
...
+DateTime             Outdoor_TimeStamp                 "Measures TimeStamp [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"         <calendar>         { channel = "netatmo:outdoor:myaccount:inside:outside:timestamp#measures" }
+DateTime             Outdoor_LastMessage               "LastMessage [%1$td.%1$tm.%1$tY %1$tH:%1$tM]"                <text>             { channel = "netatmo:outdoor:myaccount:inside:outside:timestamp#last-seen" }
+
+// Rain Module
+Number:Speed         Rain_Intensity                    "Rain Intensity [%.1f %unit%]"                               <rain>             { channel = "netatmo:rain:myaccount:inside:rainModule:rain#value"}
+Number:Length        Rain_Hour                         "Rain Last Hour [%.1f %unit%]"                               <rain>             { channel = "netatmo:rain:myaccount:inside:rainModule:rain#sum-1"}
+Number:Length        Rain_Today                        "Rain Today [%.1f %unit%]"                                   <rain>             { channel = "netatmo:rain:myaccount:inside:rainModule:rain#sum-24"}
+
+// Caméra
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/README.md:716]: Please translate to English.

File: bundles/org.openhab.binding.netatmo/README.md [hunk 2]
```diff
@@ -652,72 +652,83 @@ All these channels are read only.
...
+String               CameraSDCardState                 "SD Card State [%s]"                                         <none>             { channel = "netatmo:welcome:myaccount:myhome:mycam:status#sd-card" }
+Image                CameraLiveSnapshot                "Live Snapshot"                                              <none>             { channel = "netatmo:welcome:myaccount:myhome:mycam:live#picture" }
+String               CameraLiveStreamLocalUrl          "Live Video Stream [%s]"                                     <none>             { channel = "netatmo:welcome:myaccount:myhome:mycam:live#local-stream-url" }
+String               CameraLiveStreamVpnUrl            "Live Video Stream [%s]"                                     <none>             { channel = "netatmo:welcome:myaccount:myhome:mycam:live#vpn-stream-url" }
+Number               CameraWifiStrength                "Wifi Strength [%s]"                                         <wifi>             { channel = "netatmo:welcome:myaccount:myhome:mycam:signal#strength" }
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/README.md:723]:
```suggestion
Number               CameraWifiStrength                "Wi-Fi Strength [%s]"                                        <wifi>             { channel = "netatmo:welcome:myaccount:myhome:mycam:signal#strength" }
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks. Minor comments added.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**